conan/tools/meson/meson.py

```diff
@@ -54,9 +56,13 @@ def build(self, target=None):
         self._conanfile.run(cmd)
 
     def install(self):
-        self.configure(reconfigure=True)  # To re-do the destination package-folder
-        meson_build_folder = self._conanfile.build_folder
-        cmd = 'meson install -C "{}"'.format(meson_build_folder)
+        meson_build_folder = self._conanfile.build_folder.replace("\\", "/")
+        meson_package_folder = self._conanfile.package_folder.replace("\\", "/")
+        # TODO: Use --destdir [package_folder] param when requiring meson >= 0.57.
```

Comment:
Why not use ``--destdir`` then as argument? I think we can assume safely Meson > 0.57